bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImpl.java

```diff
@@ -14,14 +14,16 @@
 
 package com.adobe.cq.commerce.core.components.internal.models.v1.productlist;
 
-import java.util.ArrayList;
```

Comment:
Why all this import re-org?

Reply:
this is done by IDE optimize import, as discussed, we will use IDE default in sequencing import classes